### DISCUSSION TITLE
Github Topic(s) List Criteria

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24012] [New to GitHub]
<p>I believe it would be quite a help for the GitHub community if we were able to see what criteria is used when publishing a <a href="https://github.com/topics" rel="noopener nofollow ugc">topic</a>. This will not only be useful for bigger projects to draw more attention easily, but also prevent any overflow of less common topics from being on the list. Of course, the said list should be posted somewhere on the page (probably the bottom) to give content creators a heads up.</p>
<p>For starters, here are some subjective requirements:</p>
<ul>
<li><em>Certain amount of stars</em></li>
<li><em>Level of relevance</em></li>
<li><em>Is it an organization or valued subject?</em></li>
</ul>
<p>Maybe this suggestion will merge with the “<a href="https://github.com/collections" rel="noopener nofollow ugc">collections</a>” page - who knows?</p>
<p><em>Also, it would be kinda cool if there were icons for every topic instead of the pound symbol. (:</em></p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hello <a class="mention" href="/f1nx">@F1nx</a> :wave:<br>
Thanks for this feedback!</p>
<p>Here’s some information about Topics from our help docs.</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="15" height="15">
      <a href="https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics" target="_blank" rel="noopener">Classifying your repository with topics - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics" target="_blank" rel="noopener">//docs.github.com/en/github/administering-a-repository/classifying-your-repository-with-topics</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>As long as your repository has a topic properly added to it, you will appear on the list for that specific topic. We have a <a href="https://github.com/github/explore">GitHub/Explore</a> repository where you can contribute and suggest edits to an existing topic page or collection.</p>
<p>That being said, we’re always working to improve GitHub and the GitHub Support Community, and we consider every suggestion we receive.<br>
You can do so by <a href="https://support.github.com/contact/feedback">submitting this through our official product feedback form</a> so that our product team can track your request.</p>
<p>You can also keep an eye on our <a href="https://github.blog/changelog/">Changelog</a> as well as the <a href="https://github.com/github/roadmap">GitHub public roadmap</a> to see features we are working on.</p>

**Comment by ghost** [comment]
<p>And thank you for the helpful response <a class="mention" href="/ernest-phillips">@ernest-phillips</a>!</p>
<p>I went ahead and gave <code>github/explore</code> a look, and although they do give <a href="https://github.com/github/explore/blob/9e309e68e8dfc339723c559b2777083227dfb7c3/topics-todo.md" rel="noopener nofollow ugc">criteria for any topic proposed</a>, there doesn’t seem to be any kind of “final say” for what will actually be shown on the display page.</p>
<p>Am I missing something?</p>
